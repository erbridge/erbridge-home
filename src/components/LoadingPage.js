import PropTypes from 'prop-types';
import React, { Component } from 'react';

import Logo from './Logo';

import './LoadingPage.css';

class LoadingPage extends Component {
  static propTypes = {
    pointerPosition: PropTypes.shape({
      x: PropTypes.number.isRequired,
      y: PropTypes.number.isRequired,
    }),
  };

  render() {
    const { pointerPosition } = this.props;

    return (
      <div className="LoadingPage">
        <Logo
          className="LoadingPage__logo"
          focused
          pointerPosition={pointerPosition}
        />
      </div>
    );
  }
}

export default LoadingPage;
