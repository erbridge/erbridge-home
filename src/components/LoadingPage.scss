.LoadingPage {
  display: flex;
  justify-content: center;
}

.LoadingPage__logo {
  max-width: 100%;
  padding: 30px;
}
