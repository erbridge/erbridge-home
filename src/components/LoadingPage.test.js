import { shallow } from 'enzyme';
import React from 'react';

import LoadingPage from './LoadingPage';

it('renders correctly', () => {
  expect(shallow(<LoadingPage />)).toMatchSnapshot();
});
