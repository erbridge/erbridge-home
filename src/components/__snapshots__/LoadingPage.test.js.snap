// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="LoadingPage"
>
  <Logo
    className="LoadingPage__logo"
    focused={true}
  />
</div>
`;
